frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -104,7 +104,11 @@ export const FormFields = ({
   return (
     <FormProvider {...formMethods}>
       <form onSubmit={formMethods.handleSubmit(onSubmit)} noValidate>
-        <Box bg="white" py="2.5rem" px={{ base: '1rem', md: '2.5rem' }}>
+        <Box
```

Comment:
Should the box even be there at all if `formFields` is empty @karrui ?

Reply:
will just merge in first, let pearly decide when we re-QA